### PR TITLE
Change to MPLv2 license

### PR DESCRIPTION
**Pull request by anders-kiaer**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,7 +21,7 @@
 
 TESTS_REQUIRE = [
     "bandit",
-    "black>=19.10b0",
+    "black",
```
Comment (asnyv) [inline setup.py:24]: intentional?

File: src/lib/components/LeafletMap/README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (C) 2020 - Equinor ASA.
+
```
Comment (asnyv) [inline src/lib/components/LeafletMap/README.md:6]: This one renders weirdly in Markdown. Not sure if you want it to be included in the rendered README or not?
See: https://github.com/anders-kiaer/webviz-subsurface-components/tree/change_license/src/lib/components/LeafletMap/webgl/eqGL

File: src/lib/components/LeafletMap/webgl/eqGL/README.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (asnyv) [inline src/lib/components/LeafletMap/webgl/eqGL/README.md:1]: same rendering issue as mentioned before

**Review by asnyv — COMMENTED** [review]
What did we end up with for the e.g. the .json-files (like test-data and similar?) do we need to add license files to those folders?

Also, a few files at the othermost layer without license comment: https://github.com/anders-kiaer/webviz-subsurface-components/tree/change_license

I am no specialist, not sure if it is ok....

**Review by HansKallekleiv — APPROVED** [review]

**Pull request merged**